dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -15,23 +15,18 @@ aws s3 cp s3://edx-dbt-docs/manifest.json ${WORKSPACE}/manifest
 # Setup to run dbt commands
 cd $WORKSPACE/warehouse-transforms
 
-# Pull the origin master code to latest branch which will be used to compare the diff
-git pull -f origin master:latest
+BRANCH_POINT=$(git merge-base origin/master ${ghprbActualCommit})
```

Comment:
As far as I have tested so far this git merge-base approach is determining the correct delta. Yet few days of testing with actual PRs will help us determine the exact results